File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Feedbacks/Feedback.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Feedbacks;

class Feedback  implements IFeedback
{

    public $Customers;//CustomerBasic
    public $FeedbackTypes;//FeedbackTypes class tipinde
    public $ID;
    public $Title;
    public $Name;
    public $State;
    public $OrderNo;
    public $Description;
}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Feedbacks/Feedback.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Customers;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Customers;
using Ohtap.ECommerce.Domain.Models.Objects.Feedbacks;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Feedbacks
{
    public class Feedback : FeedbackBasic, IFeedback
    {
        public ICustomerData Customer { get; set; }
    }
}
